install_config/router/default_haproxy_router.adoc

```diff
@@ -1531,10 +1550,10 @@ metadata:
     prometheus.openshift.io/username: admin
 ----
 ====
-
++
```

Comment:
@knobunc didn't we change authorization to not pass the username and password as annotations?

Reply:
@lihongan  Could you review this PR?